src/templates/partials/awselb.hbs

```diff
@@ -1,86 +1,51 @@
-{
-    "AWSTemplateFormatVersion": "2010-09-09",
-    "Description": "Generated {{output.date}}, {{{output.link}}}",
-    "Parameters": {
-        "SSLCertificateId": {
-            "Description": "The ARN of the SSL certificate to use",
-            "Type": "String",
-            "AllowedPattern": "^arn:[^:]*:[^:]*:[^:]*:[^:]*:.*$",
-            "ConstraintDescription": "SSL Certificate ID must be a valid ARN. http://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#genref-arns"
-        }
-    },
-    "Resources": {
-        "ExampleELB": {
-            "Type": "AWS::ElasticLoadBalancing::LoadBalancer",
-            "Properties": {
-                "Listeners": [
-                    {
-                        "LoadBalancerPort": "443",
-                        "InstancePort": "80",
-                        "PolicyNames": [
-                            "Mozilla-{{form.config}}-2015-03"
-                        ],
-                        "SSLCertificateId": {
-                            "Ref": "SSLCertificateId"
-                        },
-                        "Protocol": "HTTPS"
-                    }
-                ],
-                "AvailabilityZones": {
-                    "Fn::GetAZs": ""
-                },
-                "Policies": [
-                    {
-                        "PolicyName": "Mozilla-{{form.config}}-2015-03",
-                        "PolicyType": "SSLNegotiationPolicyType",
-                        "Attributes": [
+AWSTemplateFormatVersion: 2010-09-09
+Description: Mozilla ELB configuration generated {{output.date}}, {{{output.link}}}
+Parameters:
+  SSLCertificateId:
+    Description: The ARN of the ACM SSL certificate to use
+    Type: String
+    AllowedPattern: ^arn:aws:acm:[^:]*:[^:]*:certificate/.*$
+    ConstraintDescription: >
+      SSL Certificate ID must be a valid ACM ARN.
+      https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#genref-arns
+Resources:
+  ExampleELB:
+    Type: AWS::ElasticLoadBalancing::LoadBalancer
+    Properties:
+      Listeners:
+        - LoadBalancerPort: '443'
+          InstancePort: '80'
+          PolicyNames:
+            - Mozilla-{{form.config}}-v5-0
```

Comment:
You can use `{{sstls.version}}` here, which will return 5, 5.1, 6, etc.

Reply:
Unfortunately in this specific case, AWS SSL policy names aren't allowed to have the `.` character so we could
* leave it hard coded
* produce an additional variable, in addition to `sstls.version` but which doesn't user periods
* do something here in the template to convert `.` to `-` (maybe we load a handlebar helper)?
* something I'm not thinking of.

What direction do you think we should go?

Reply:
We chatted and we're going to leave it like it is for now